data/campaigns/Under_the_Burning_Suns/utils/macros.cfg

```diff
@@ -134,3 +134,36 @@ if(red <= green and green <= blue + 3, ({NEWCHAN} / 255.0)^2 * 255.0, {ORIGCHAN}
         [/object]
     [/event]
 #enddef
+
+#define UTBS_XP_MODIFIER
+    experience_modifier={ON_DIFFICULTY 100 150 200}
+#enddef
+
+#define UTBS_CORRECT_RECALL_COST EVENT
+    [event]
+        name = {EVENT}
```

Comment:
It doesn't look like there's any reason for this to be a parameter? Every instance passes `start`.

Reply:
Oversight